### PR TITLE
PR for finshed mobile version protifolio

### PR DESCRIPTION
**Pull request by Tekhlay**
#This is the complete version of the mobile responsive portfolio
## Core recaps in this version
- Works section is added to the index.html file
- about section is added to the index.html file
- The footer section is added to the index.html file
- The CSS file is deeply updated and feer of any errors now

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,8 +41,170 @@ <h1>Hey there. I’m Mariane <span class="h-span">I’m a software developer</sp
             </button>
         </div>
     </section>
-    <div class="myrecentwork">
-        <h1>My Recent Works</h1>
-    </div>
+    <section>
```
Comment (1GORDON) [inline index.html:44]: Great job @Tekhlay 🤝

- [ ] As stated in the project requirements, kindly align the elements of the `My works section` with `display: grid`.

File: index.html [hunk 2]
```diff
@@ -41,8 +41,170 @@ <h1>Hey there. I’m Mariane <span class="h-span">I’m a software developer</sp
...
+                <ul class="languages">
+                    <li><a href="">Ruby on rails</a></li>
+                    <li><a href="">css</a></li>
+                    <li><a href="">JavaScript</a></li>
+                    <li><a href="">html</a></li>
+                </ul>
+                <button class="see-project" type="button">
+                    <span class="project-title">See Project</span> </button>
+            </div>
+        </div>
```
Comment (1GORDON) [inline index.html:78]: Great job @Tekhlay 🤝

- [ ] As stated in the project requirements, kindly align the elements of these individual cards with `display: flex`.

File: index.html [hunk 3]
```diff
@@ -41,8 +41,170 @@ <h1>Hey there. I’m Mariane <span class="h-span">I’m a software developer</sp
...
+            </div>
+            <div class="txt-info">
+                <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="languages">
+                    <li><a href="">Ruby on rails</a></li>
+                    <li><a href="">css</a></li>
+                    <li><a href="">JavaScript</a></li>
+                    <li><a href="">html</a></li>
+                </ul>
+                <button class="see-project" type="button">
```
Comment (1GORDON) [inline index.html:107]: Great job @Tekhlay 🤝

- [ ] Kindly implement the button interactions of `hover`, `active`, and `enable` across all your buttons.

File: index.html [hunk 4]
```diff
@@ -41,8 +41,170 @@ <h1>Hey there. I’m Mariane <span class="h-span">I’m a software developer</sp
...
+                    <span class="project-title">See Project</span> </button>
+            </div>
+        </div>
+    </section>
+    <section class="abt">
```
Comment (1GORDON) [inline index.html:144]: Great job @Tekhlay 🤝

- [ ] Kindly align the contents of the `About Me Section` with `display: flex`.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Tekhlay  ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Linters are passing ✔️
- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions

- [ ] You may always write your commit messages using an imperative tense something like `Add Linter Files` instead of `Added Linter Files`.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 5]
```diff
@@ -150,3 +150,344 @@ a {
...
+  height: 97px;
+  padding: 22px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  position: relative;
+  display: grid;
+  align-items: center;
```
Comment (christianbiring1) [inline css/style.css:165]: - [ ] Kindly make sure your page is not overlapping as it is. It is supposed to fit 100% of the screen size and not more. as you can see in the provided Figma template
- [ ] Also you may consider using the same `typo(font-size, font family)` and `color` as the ones used in the Figma template.
Kindly fix that and you will be good to go.
Your design :point_down: 

image
image
 
Figma design :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Tekhlay , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
# REQUEST CHANGES :recycle: 
soclose

 ## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 5 above)
Comment (Tekhlay) [reply css/style.css:165]: Hello @christianbiring1 
## Actually I do not get you. kindly please would you explain it more. as I have attached in below my design looks fin and overlapped.  
image

**Review by frank1738 — APPROVED** [review]
Hi @Tekhlay  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**